apps/bare-expo/ios/Podfile.lock

```diff
@@ -1194,7 +1194,7 @@ SPEC CHECKSUMS:
   EXStoreReview: 40674cc897a6d7fd249969b86d1833f67b99170a
   EXStructuredHeaders: 384ccdf0c09ed0be8fc73d4b2d70e436cd195975
   EXTaskManager: 91fef764a2ec7690aa8070f862098df064facdc4
-  EXTrackingTransparency: 3632a570eef235f579c2e004f6515ba352bc735f
+  EXTrackingTransparency: dc21f285b53a1780e77d40fd888573f50615ccfc
```

Comment:
Hmm, not sure if this needs to be updated 🤔 I've merged some changes there a while ago 🤔